runtime_scan/pkg/orchestrator/scanwatcher/helpers.go

```diff
@@ -28,9 +28,17 @@ func isScanTimedOut(scan *models.Scan, timeout time.Duration) bool {
 	if scan == nil || scan.StartTime == nil {
 		return false
 	}
-	deadline := (*scan.StartTime).Add(timeout)
 
-	return time.Now().UTC().After(deadline)
+	now := time.Now().UTC()
+	// Use the provided timeout to calculate the timeoutTime by default.
+	timeoutTime := scan.StartTime.Add(timeout)
```

Comment:
I think it would be more readable to switch this to use the DefaultScanTimeout const here instead of passing  timeout into isScanTimedOut. Its just one more level of indirection which makes this hard to understand. WDYT?

Reply:
There are two reasons why we pass the timeout parameter here:
* the default timeout parameter is comes from the `scanwatcher.Config` which can be set by `SCAN_TIMEOUT` which has the `DefaultScanTimeout` as a default.
* the `isScanTimedOut` is a helper function which is not method of the `ScanWatcher` and it should not be

I am leaning towards make the `isScanTimedOut` as a method for `models.Scan` with the same semantics.

Reply:
hmm, I see, I didn't realise the default was configurable through the scanwatcher.Config. I agree moving it as a helper function on the Scan makes sense. Also I think we need to change the variable naming a bit to make it obvious that its the default timeout being passed into the function:

```
func (s *Scan) IsTimedOut(defaultTimeout time.Duration) {
   timeoutDuration := defaultTimeout
   if timeoutSeconds := s.GetTimeoutSeconds(); timeout > 0 {
       timeoutDuration = time.Duration(timeoutSeconds) * time.Second
   }
   
   endTime := s.StartTime.Add(timeoutDuration)
   return time.Now().After(endTime)
}
```